eng/pipelines/runtimelab/runtimelab-post-build-steps.yml

```diff
@@ -30,12 +32,22 @@ steps:
     - ${{ if eq(parameters.platform, 'browser_wasm_win') }}:
       - script: |
           call $(Build.SourcesDirectory)\wasm-tools\emsdk\emsdk_env
-          $(Build.SourcesDirectory)/src/tests/build$(scriptExt) nativeaot $(buildConfigUpper) ${{ parameters.archType }} tree nativeaot /p:LibrariesConfiguration=${{ parameters.librariesConfiguration }}
+          $(Build.SourcesDirectory)/src/tests/build$(scriptExt) nativeaot $(buildConfigUpper) ${{ parameters.archType }} tree nativeaot /p:LibrariesConfiguration=${{ parameters.librariesConfiguration }} /p:TestWrapperTargetsWindows=true
         displayName: Build WebAssembly tests
     - ${{ elseif eq(parameters.platform, 'wasi_wasm_win') }}:
       - script: |
           call $(Build.SourcesDirectory)\wasm-tools\emsdk\emsdk_env
-          $(Build.SourcesDirectory)/src/tests/build$(scriptExt) nativeaot $(buildConfigUpper) ${{ parameters.archType }} wasi tree nativeaot /p:LibrariesConfiguration=${{ parameters.librariesConfiguration }}
+          $(Build.SourcesDirectory)/src/tests/build$(scriptExt) nativeaot $(buildConfigUpper) ${{ parameters.archType }} wasi tree nativeaot /p:LibrariesConfiguration=${{ parameters.librariesConfiguration }} /p:TestWrapperTargetsWindows=true
+        displayName: Build WebAssembly tests
+    - ${{ elseif eq(parameters.platform, 'Browser_wasm_linux_x64') }}:
+      - script: |
+          source $(Build.SourcesDirectory)/wasm-tools/emsdk/emsdk_env.sh
+          LD_LIBRARY_PATH=/usr/local/lib:/opt/lib:/lib:/usr/lib:/crossrootfs/x64/lib/x86_64-linux-gnu/ $(Build.SourcesDirectory)/src/tests/build$(scriptExt) nativeaot $(buildConfigUpper) -arch:${{ parameters.archType }} -target-os:browser tree nativeaot /p:LibrariesConfiguration=${{ parameters.librariesConfiguration }}
+        displayName: Build WebAssembly tests
+    - ${{ elseif eq(parameters.platform, 'wasi_wasm_linux_x64') }}:
+      - script: |
+          source $(Build.SourcesDirectory)/wasm-tools/emsdk/emsdk_env.sh
+          LD_LIBRARY_PATH=/usr/local/lib:/opt/lib:/lib:/usr/lib:/crossrootfs/x64/lib/x86_64-linux-gnu/ $(Build.SourcesDirectory)/src/tests/build$(scriptExt) nativeaot $(buildConfigUpper) -arch:${{ parameters.archType }} -target-os:wasi tree nativeaot /p:LibrariesConfiguration=${{ parameters.librariesConfiguration }}
```

Comment:
Why is a `LD_LIBRARY_PATH` like this needed?

Reply:
Because `libtinfo.so.5` is only in the rootfs.  The host has `libtinfo.so.6`

Reply:
Do we build the LLVM with libtinfo dependency ourselves or is that for pre-built LLVM?

If we build it ourselves, we can get rid of libtinfo dependency like this: https://github.com/dotnet/runtimelab/pull/741/files#diff-8bb555c9ef0d91e66546d2fa904a310f4d22f129e191dff6b12e4fbdfe65e321R33